### DISCUSSION TITLE
Are the rules for mysql updated removed from the latest version?

### DISCUSSION THREAD

**Discussion by 7thstorm** [rectorphp/rector#8158] [Q&A] [answered]
I cannot find these rules:
   $rectorConfig->rule(\Rector\MysqlToMysqli\Rector\Assign\MysqlAssignToMysqliRector::class);
    $rectorConfig->rule(\Rector\MysqlToMysqli\Rector\FuncCall\MysqlFuncCallToMysqliRector::class);
    $rectorConfig->rule(\Rector\MysqlToMysqli\Rector\FuncCall\MysqlPConnectToMysqliConnectRector::class);
    $rectorConfig->rule(\Rector\MysqlToMysqli\Rector\FuncCall\MysqlQueryMysqlErrorWithLinkRector::class);

I used them in ver .17x but don't see them anymore

**Comment by samsonasik** [comment, marked as answer]
They are removed, see PR:

- https://github.com/rectorphp/rector-src/pull/4448

for the reasoning.
